### PR TITLE
Add tests to Github actions

### PR DESCRIPTION
**Pull request by aramboi**
Uses a SQLite database in the test configuration.

### CODE REVIEW

File: .github/workflows/ci_api.yml [hunk 1]
```diff
@@ -13,18 +13,21 @@ on:
...
       run: |
         python -m pip install --upgrade pip
-        pip install black
+        pip install -r ./api/requirements-dev.txt
     - name: Check formatting with black
       run: |
         black --line-length 80 --target-version py37 --exclude migrations --check ./api
+    - name: Run pytest
+      run: |
+        pytest
```
Comment (tudoramariei) [inline .github/workflows/ci_api.yml]: I'm still not sure if it's better to have tests run together with formatting
I feel that leaving them separate would be better
- we won't need to worry that one is holding up the other 
- we can have better customizability if we want to run them under different constraints
- we're going to see one failure and no success if either of them fails

**Comment by tudoramariei** [comment]
I'd also ask you to change the title from the current `CI` to something more relevant like `Run formatting check and tests` (if the code remains in the current state).

File: .github/workflows/ci_api.yml (hunk 1 above)
Comment (aramboi) [reply .github/workflows/ci_api.yml]: You are right. We could keep the same file and run parallel jobs I think.

**Comment by aramboi** [comment]
Made the requested changes and rebased. I think this is ready to merge.

**Comment by tudoramariei** [comment]
Just to place the proper connections in order
This PR is related to #49

**Pull request merged**